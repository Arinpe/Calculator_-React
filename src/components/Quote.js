const Quote = () => (
  <p className="quote">
    “Common sense is calculation applied to life. ”
    ― Henri Frederic Amiel
  </p>
);
export default Quote;
