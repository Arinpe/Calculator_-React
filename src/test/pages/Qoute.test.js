import React from 'react';
import render from 'react-test-renderer';
import Quote from '../../components/Quote';

describe('Quote', () => {
  const page = render.create(<Quote />).toJSON();
  test('should render without crashing', () => {
    expect(page).toMatchSnapshot();
  });
});
