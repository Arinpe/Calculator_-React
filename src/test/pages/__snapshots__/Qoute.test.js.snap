// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote should render without crashing 1`] = `
<p
  className="quote"
>
  “In real life, I assure you, there is no such thing as algebra. ” ― Fran Lebowitz
</p>
`;
